cmake_ext.py

```diff
@@ -7,18 +7,14 @@
 
 
 class CMakeExtension(Extension):
-    def __init__(self, sourcedir=""):
-        Extension.__init__(self, "CMakeExtension", sources=[])
+    def __init__(self, name, sourcedir=""):
+        Extension.__init__(self, name, sources=[])
         self.sourcedir = os.path.abspath(sourcedir)
 
 
 class CMakeBuild(build_ext):
-    def run(self):
-        for ext in self.extensions:
-            self.build_extension(ext)
-
     def build_extension(self, ext):
-        extdir = Path(self.get_ext_fullpath(ext.name)).parent.absolute() / "iminuit"
+        extdir = Path(self.get_ext_fullpath(ext.name)).parent.absolute()
```

Comment:
If you include a dot in the name, you don't have to fiddle with the path here. And it works in the check functions too!